### ISSUE TITLE
Unusually high number of mach ports

### ISSUE THREAD

**Issue by pandalion98** [crystalidea/macs-fan-control#185]
It seems like `Ports` increases by 1 every second. It resets when the program is relaunched.
<img width="427" alt="Screen Shot 2019-06-28 at 02 01 17" src="https://user-images.githubusercontent.com/4461705/60289284-a49da280-9948-11e9-9237-0768175524be.png">

And according to [this link](https://apple.stackexchange.com/q/212277/230768), leaking ports isn't a good thing. Hopefully this is resolved soon.

---
Required stuff according to CONTRIBUTING.md:
<img width="826" alt="Screen Shot 2019-06-28 at 01 50 14" src="https://user-images.githubusercontent.com/4461705/60289046-1f19f280-9948-11e9-9a47-7c89c90c8f24.png">

```
Macs Fan Control 1.4.12
MacBookPro12,1
CPU: Intel(R) Core(TM) x CPU @ x [100]
GPU: Intel Iris Graphics x
macOS 10.14.5

Fans:
Right Side-min{1299}curr{3454}max{6199}mode{2,TCXC,42,65}


TempSensors:
TA0P (Ambient) - 35.75, 35.75
TB0T (Battery Max) - 32, 32
TB1T (Battery Sensor 1) - 32, 32
TB2T (Battery Sensor 2) - 30.8984, 30.8984
TBXT () - 32, 32
TC0C (CPU Core 1) - 47, 47
TC0E () - 56.2227, 56.2227
TC0F () - 57.625, 57.625
TC0P (CPU Proximity) - 45.75, 45.75
TC1C (CPU Core 2) - 48, 48
TCGC (GPU PECI) - 48, 48
TCSA () - 49, 49
TCXC (CPU PECI) - 49.6562, 49.6562
TH0A () - 36.875, 36.875
TH0B () - 36.25, 36.25
TH0C () - 35.125, 35.125
TH0F () - 207.082, 207.082
TH0R () - 207.125, 207.125
TH0V () - 35.1602, 35.1602
TH0a () - 36.875, 36.875
TH0b () - 36.25, 36.25
TH0c () - 35.125, 35.125
TH0x () - 35.125, 35.125
THSP () - 129, 129
TM0P (Memory Proximity) - 41.875, 41.875
TMLB () - 129, 129
TPCD (Platform Controller Hub Die) - 45, 45
TW0P () - 129, 129
Th1H (Heatsink Proximity) - 38.625, 38.625
Ts0P (Palm Rest) - 31.875, 31.875
Ts0S () - 36.6406, 36.6406
Ts1P () - 31.25, 31.25
Ts1S () - 35.6172, 35.6172
Ts2S () - 34.1133, 34.1133


Disk drives:
APPLE SSD SM0128G - 34.875

```

**Comment by traisjames** [comment]
I am finding the same thing too.  Didn't even look until I saw you mentioned this.  Here is some potentially useful files and screenshots from Activity Manager.
[Sample of MacsFanControl.txt](https://github.com/crystalidea/macs-fan-control/files/3457896/Sample.of.MacsFanControl.txt)

Ports and Files page:

`cwd
/
txt
/Applications/Macs Fan Control.app/Contents/MacOS/MacsFanControl
txt
/Applications/Macs Fan Control.app/Contents/Frameworks/QtWidgets.framework/Versions/5/QtWidgets
txt
/Applications/Macs Fan Control.app/Contents/Frameworks/QtGui.framework/Versions/5/QtGui
txt
/Applications/Macs Fan Control.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore
txt
/Applications/Macs Fan Control.app/Contents/Frameworks/QtMacExtras.framework/Versions/5/QtMacExtras
txt
/Applications/Macs Fan Control.app/Contents/Frameworks/QtNetwork.framework/Versions/5/QtNetwork
txt
/Applications/Macs Fan Control.app/Contents/Frameworks/QtConcurrent.framework/Versions/5/QtConcurrent
txt
/Library/Preferences/Logging/.plist-cache.l36HFtro
txt
/Applications/Macs Fan Control.app/Contents/Frameworks/QtPrintSupport.framework/Versions/5/QtPrintSupport
txt
/private/var/db/timezone/tz/2019b.1.0/icutz/icutz44l.dat
txt
/usr/lib/libobjc-trampolines.dylib
txt
/usr/share/icu/icudt62l.dat
txt
/Applications/Macs Fan Control.app/Contents/PlugIns/platforms/libqcocoa.dylib
txt
/System/Library/CoreServices/SystemAppearance.bundle/Contents/Resources/SystemAppearance.car
txt
/System/Library/Fonts/Helvetica.ttc
txt
/System/Library/Keyboard Layouts/AppleKeyboardLayouts.bundle/Contents/Resources/AppleKeyboardLayouts-L.dat
txt
/System/Library/CoreServices/SystemAppearance.bundle/Contents/Resources/DarkAquaAppearance.car
txt
/System/Library/Fonts/Monaco.dfont
txt
/usr/lib/dyld
txt
/System/Library/CoreServices/SystemAppearance.bundle/Contents/Resources/DarkAppearance.car
txt
/System/Library/CoreServices/SystemAppearance.bundle/Contents/Resources/Assets.car
txt
/System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/HIToolbox.framework/Versions/A/Resources/Extras2.rsrc
txt
/System/Library/Fonts/SFNSText.ttf
txt
/System/Library/CoreServices/SystemAppearance.bundle/Contents/Resources/VibrantLightAppearance.car
txt
/Library/Application Support/CrashReporter/SubmitDiagInfo.domains
txt
/private/var/folders/q2/c1g1qc4d56gccqw7pv1_tl3r0000gn/C/com.crystalidea.MacsFanControl/com.apple.metal/3902/libraries.data
txt
/private/var/folders/q2/c1g1qc4d56gccqw7pv1_tl3r0000gn/C/com.crystalidea.MacsFanControl/com.apple.metal/Intel Iris Pro Graphics/functions.data
0
/dev/null
1
/dev/null
2
/dev/null
3
/var/folders/q2/c1g1qc4d56gccqw7pv1_tl3r0000gn/T/MACSFANCONTROL
4
->0x725498e1077c767d
5
->0x725498e1077c78d5
6
/Library/Application Support/CrashReporter/SubmitDiagInfo.domains
7
->0x725498e102e6c475
8
->0x725498e102e6c235
9
/System/Library/Frameworks/CoreImage.framework/Versions/A/Resources/ci_stdlib.metallib
10
/System/Library/Frameworks/CoreImage.framework/Versions/A/Resources/ci_filters.metallib
11
/private/var/folders/q2/c1g1qc4d56gccqw7pv1_tl3r0000gn/C/com.crystalidea.MacsFanControl/com.apple.metal/3902/libraries.maps
12
/private/var/folders/q2/c1g1qc4d56gccqw7pv1_tl3r0000gn/C/com.crystalidea.MacsFanControl/com.apple.metal/3902/libraries.data
13
/private/var/folders/q2/c1g1qc4d56gccqw7pv1_tl3r0000gn/C/com.crystalidea.MacsFanControl/com.apple.metal/Intel Iris Pro Graphics/functions.maps
14
/private/var/folders/q2/c1g1qc4d56gccqw7pv1_tl3r0000gn/C/com.crystalidea.MacsFanControl/com.apple.metal/Intel Iris Pro Graphics/functions.data
15
->0x725498e102e6aa35
16
->0x725498e102e6c535
19
->0x725498e102e6aeb5
20
->0x725498e102e6a4f5
25
26
->0x725498e1077c6f75
27
[ctl com.apple.netsrc id 8 unit 29]
`
<img width="364" alt="Screen Shot 2019-08-01 at 12 14 24 PM" src="https://user-images.githubusercontent.com/751219/62313424-58b5b000-b456-11e9-98a5-4d69e00a9bf3.png">
<img width="441" alt="Screen Shot 2019-08-01 at 12 14 33 PM" src="https://user-images.githubusercontent.com/751219/62313425-58b5b000-b456-11e9-8bc8-b4155ecc1020.png">

**Comment by kleuter** [comment]
Must be fixed now in v1.5
